shared-libs/user-management/src/users.js

```diff
@@ -582,9 +561,14 @@ const missingFields = data => {
 const getUpdatedUserDoc = async (username, data, fullAccess) => {
   return getUserDoc(username, 'users')
     .then(async doc => {
-      return {
+      const baseUser = {
         ...doc,
-        ...(await getUserUpdates(username, data, fullAccess)),
+        name: username,
+        type: 'user',
+      };
```

Comment:
suggestion: I don't think we actually need this here.  `doc` will always have `name` and `type`.  I have tried manually creating docs in the `_users` db in Couch and it rejects anything without a `name` or when `type != 'user'`.

Reply:
That's true. In production it would not be necessary, but the tests were breaking because the mocks in the test for the doc were empty objects `db.users.get.resolves({})`. 

I've removed the baseUser which was setting the name and updated the mocks to be more realistic

```
db.users.get.resolves({
  name: 'paul',
  type: 'user'
});
```